test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,38 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest"
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
+		framework.ExpectNoError(err, "failed creating Namespace")
+
+		ginkgo.By("patching the Namespace")
+		nspatch := fmt.Sprintf(`{"metadata":{"labels":{"testProperty":"testValue"}}}`, namespaceName)
+		_, err = f.ClientSet.CoreV1().Namespaces().Patch(namespaceName, types.StrategicMergePatchType, []byte(nspatch))
+		framework.ExpectNoError(err, "failed to patch Namespace")
+
+		ginkgo.By("finding the namespace with it's LabelSelector")
```

Comment:
`it's` -> `its`

Reply:
Thanks, I've update this in f13b448bc2db1